src/pages/UsersPage/UsersPage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { Button } from '../../components/Button';
+import { ButtonTypes } from '../../components/Button/interfaces';
+import { UserPageProps } from './interfaces';
+import './usersPage.scss';
+
+export const UsersPage: React.FC<UserPageProps> = ({
+  users,
+  handleDelete,
+  toggleChecked,
+  changeSetIsAuth,
+}) => {
+  return (
+    <div className="user-page">
+      <Button
+        btnClass="send sign-out__btn"
+        btnText="Sign out"
+        btnType={ButtonTypes.button}
+        onClick={() => changeSetIsAuth(false)}
+      />
```

Comment:
I don't see this button in the requirements

Reply:
deleted